x-pack/plugins/synthetics/server/routes/filters/filters.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
+import { syntheticsMonitorType } from '../../../common/types/saved_objects';
+import { ConfigKey } from '../../../common/runtime_types';
+import { SYNTHETICS_API_URLS } from '../../../common/constants';
+import { SyntheticsRestApiRouteFactory } from '../../legacy_uptime/routes/types';
+
+type Buckets = Array<{
+  key: string;
+  doc_count: number;
+}>;
+
+interface AggsResponse {
+  monitorTypes: {
+    buckets: Buckets;
+  };
+  locations: {
+    buckets: Buckets;
+  };
+  tags: {
+    buckets: Buckets;
+  };
+  projects: {
+    buckets: Buckets;
+  };
+  schedules: {
+    buckets: Buckets;
+  };
+}
+
+export const getSyntheticsFilters: SyntheticsRestApiRouteFactory = () => ({
+  method: 'GET',
+  path: SYNTHETICS_API_URLS.FILTERS,
+  validate: {},
+  handler: async ({ savedObjectsClient, request, response, server }): Promise<any> => {
+    try {
+      await server.spaces?.spacesService.getActiveSpace(request);
```

Comment:
This can be removed since calling this api with an invalid space would simply result in all values being empty.